lib/Froxlor/Cron/System/MailboxsizeCron.php

```diff
@@ -38,26 +38,37 @@ public static function run()
 			$_maildir = \Froxlor\FileDir::makeCorrectDir($maildir['maildirpath']);
 
 			if (file_exists($_maildir) && is_dir($_maildir)) {
-				// mail-address allows many special characters, see http://en.wikipedia.org/wiki/Email_address#Local_part
-				$return = false;
-				$back = \Froxlor\FileDir::safe_exec('du -sk ' . escapeshellarg($_maildir), $return, array(
-					'|',
-					'&',
-					'`',
-					'$',
-					'~',
-					'?'
-				));
-				foreach ($back as $backrow) {
-					$emailusage = explode(' ', $backrow);
-				}
-				$emailusage = floatval($emailusage['0']);
+				// 1. Check if maildirsize exists (usually when quota is enabled)
+				$maildirsize = $_maildir.'maildirsize';
```

Comment:
Could we change that line to the followig, just to be sure? More froxlor-style xD
```
$maildirsize = \Froxlor\FileDir::makeCorrectFile($_maildir . '/maildirsize');
```